core/math/dynamic_bvh.h

```diff
@@ -41,7 +41,7 @@
 
 /*
 Bullet Continuous Collision Detection and Physics Library
-Copyright (c) 2003-2013 Erwin Coumans  https://bulletphysics.org
+Copyright (c) 2003-2013 Erwin Coumans  http://bulletphysics.org
```

Comment:
Should this be `http://continuousphysics.com/Bullet/`?

Reply:
No, because this copyright statement was originally bulletphysics.org:

https://github.com/godotengine/godot/pull/51971/files#diff-e39a9b6fe671bb813a63a8731f7248baf6223bcaff39456948c83c51e946d958L42

This one is 2013 while the others are 2006, which may explain it. Bullet uses both interchangeably in different files, so I guess we have to preserve this.